platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -293,173 +313,157 @@ the state:
 
-```java
-//in PlatformBuilder
-
-/*
-  Set the Genesis Roster for the network.
-  <p>
-  This method is used to set the optional Genesis Roster for the network.
-  The Genesis Roster is a special roster used for the sole-purpose of bootstrapping a network.
-  The Genesis Roster, if set, is immutable and will be used to start a Genesis network Process or Network Transplant process.
-  <p>
-  @param genesisRoster the Genesis Roster for the network.
- */
-PlatformBuilder withGenesisRoster(@NonNull final Roster genesisRoster);
-```
+A new, special-purpose config.txt file will be introduced and named `genesis-config.txt`.
+On startup, the determination logic (of the correct roster to use) will check for the existence of this`genesis-config.txt` file.
+This file will be in the same location and format as the current `config.txt` file.
 
-The logic to determine whether to invoke this method could be determined by some mechanism Services deems appropriate.
+A start-up determination logic will check if there is a `Roster` present in the state.
+If not, the determination logic will look for a `genesis-config.txt` on disk.
+If found, a Genesis Roster will be created from it and used as the active `Roster` for the network.
 
 #### Override Roster
 
-An `Override Roster` is an optional Roster that DevOps may provide for the explicit purpose of starting a network using
-a specific state, with a specific set of nodes, such as during a Network Transplant Process. A common example of
-this is Mainnet State being transplanted into a testing network.
+An `Override Roster` is an optional Roster that DevOps may provide for the explicit purpose of starting a network using a specific state, with a specific set of nodes, such as during a Network Transplant Process.
+A common example of this is Mainnet State being transplanted into a testing network.
 
-A quick note on PCES. The software version is used to determine which roster (active or previous) should be used to
-validate the signatures of events in the PCES. If the PCES event version is less than the current software version,
-it indicates that the event was signed with the old roster, and hence, the old roster is used for validation. If the
-PCES event version matches the current software version, the active roster is used. This behavior will be
-maintained until Birth Rounds or Dynamic Address Book is implemented.
+A quick note on Pre-Consensus Event Stream (PCES).
+The software version is used to determine which roster (active or previous) should be used to validate the signatures of events in the PCES.
+If the PCES event version is less than the current software version, it indicates that the event was signed with the old roster, and hence, the old roster is used for validation.
+If the PCES event version matches the current software version, the active roster is used.
+This behavior will be maintained until Birth Rounds or Dynamic Address Book is implemented.
```

Comment:
I think we need to expand this section to avoid the usage of the "previous roster". Note that the "previous AddressBook" is okay as it describes the current behavior, but there's not a "previous roster" - there's just an entry in the history with an older round number.

Also, and most importantly, we need to describe those artificially added records to the history in case the old and the current roster are the same (or there's not an old roster at all.)

Reply:
Not quite accurate. The round number in the Roster history tells us at which round a particular Roster has become active. For example, the history could look like `[{100, hash1}, {200, hash2}]`, while currently we may be processing the round number 250. Per this history object, even though the hash2 has a round number that is less than the current round number, the Roster with `hash2` is still considered active as of now.

Regardless though, the updated wording in the document looks fine. Thanks!

Reply:
I think my use of the word "consensus round" is inaccurate. The point stands though that there's an existence of a "previous roster", and it is the roster that was used to sign events in rounds older than the starting round number of the active roster. One may or may not be present in the state. It won't be present if in Genesis or in the edge cases described in this proposal (which we have a solution to artificially create one). 
I know this comment is resolved, but just adding this comment for completeness.